### PR TITLE
Temp_PR-BC_WIP

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

**Comment by NoSloppy** [comment]
This isn't right. I tested a different version. I'll be back.

**Comment by NoSloppy** [comment]
ok

File: props/BC_Working_temp_relevant_only.h [hunk 1]
```diff
@@ -28,6 +28,105 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
   SaberBCButtons() : PropBase() {}
   const char* name() override { return "SaberBCButtons"; }
 
+BLADE_ID_CLASS_INTERNAL blade_id;  // still needed?
```
Comment (profezzorn) [inline props/BC_Working_temp_relevant_only.h:31]: What is this for?

File: props/BC_Working_temp_relevant_only.h [hunk 2]
```diff
@@ -76,103 +202,123 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
...
 
-  void PrintBladesStatus() {
-    PVLOG_DEBUG << "***** Updated. Current blade on: " 
-           << (SaberBase::OnBlades()[BC_MAIN_BLADE] ? " MAIN " : "") 
-           << (SaberBase::OnBlades()[BC_SECOND_BLADE] ? " SECOND " : "") 
-           << "\n";
-  }
-
+  
   void TurnBladeOn(BladeSet target_blade) {  // receives opposite blade (MAIN or SECOND) so we can exclude it
```
Comment (profezzorn) [inline props/BC_Working_temp_relevant_only.h:214]: Is this comment still accurate?
If so, you should fix that.
If not, remove the comment.

Comment (profezzorn) [reply props/BC_Working_temp_relevant_only.h:214]: I would recommend doing this:

// Add all non-controlled blades.
target_blade = target_blade | ~controlled_blades_;

That way you don't have to feed in the "opposite blade" anymore.

File: props/BC_Working_temp_relevant_only.h (hunk 1 above)
Comment (NoSloppy) [reply props/BC_Working_temp_relevant_only.h:31]: Perfect question, and hence the comment to myself. Searching The Crucible for it, it looks like it's a remnant from the first attempt of implementing constant BladeID monitoring.
Gone now.

**Pull request closed without merging**